### PR TITLE
Store pdf results in separates folders to avoid confusions

### PR DESCRIPTION
**Pull request by SoftDrink**

### CODE REVIEW

File: wsf_scraping/pipelines.py [hunk 1]
```diff
@@ -22,24 +22,33 @@ def __init__(self):
             self.settings['SECTIONS_KEYWORDS_FILE']
         )
 
+        if not os.path.isdir('./results/pdfs'):
+            os.makedirs('./results/pdfs')
+            os.makedirs('./results/pdfs/nice')
+            os.makedirs('./results/pdfs/who_iris')
```
Comment (nsorros) [inline wsf_scraping/pipelines.py]: It might worth adding a list of spiders in the settings from which you will create the folders and check that the spider name is in the list when someone is calling process item to avoid the code trying to save in a folder that does not exist.

File: wsf_scraping/pipelines.py [hunk 2]
```diff
@@ -68,13 +77,13 @@ def process_item(self, item, spider):
         has_keywords = len(item['keywords'])
 
         # If we need to keep the pdf, move it, else delete it
-        if keep_pdf and has_keywords:
+        if keep_pdf or has_keywords:
```
Comment (nsorros) [inline wsf_scraping/pipelines.py]: why did you change to or?

File: wsf_scraping/pipelines.py (hunk 1 above)
Comment (SoftDrink) [reply wsf_scraping/pipelines.py]: Good idea! I'll do it in the next rev.

File: wsf_scraping/pipelines.py (hunk 2 above)
Comment (SoftDrink) [reply wsf_scraping/pipelines.py]: In both case we want to keep the pdf, either the keep_pdf setting is at 'True' or a keyword was matched in the article

Comment (nsorros) [reply wsf_scraping/pipelines.py]: 👍
